php-practice/php-practice_2.php

```diff
@@ -1,15 +1,111 @@
 <?php
 // Q1 tic-tac問題
+echo '1から100までのカウントを開始します。' . "\n";
+
+for ($i = 1; $i < 101; $i++) {
```

Comment:
[imo]
下記の方が100までがわかりやすいと思いましたがいかがでしょうか？
```suggestion
for ($i = 1; $i <= 100; $i++) {
```

Reply:
仰るように「<=100」と書いたほうが「100までのカウントを行う処理」と一目で分かる記述になると感じたので、
561495222045844d32bd7775c50648c116cd3e5a にて修正いたしました。ご指摘いただきありがとうございます。